nni/retiarii/trainer/pytorch/base.py

```diff
@@ -1,3 +1,5 @@
+# This file is deprecated.
```

Comment:
why not directly remove it?

Reply:
Because I don't want to remove the cgo_engine. If I remove this file and do not remove cgo_engine, linter will complain.